chia/data_layer/data_store.py

```diff
@@ -330,6 +332,8 @@ async def get_roots_between(
                 "AND generation >= :generation_begin AND generation < :generation_end ORDER BY generation ASC",
                 {"tree_id": tree_id.hex(), "generation_begin": generation_begin, "generation_end": generation_end},
             )
+            if cursor.rowcount < 1:
+                raise Exception("tree id not in store")
```

Comment:
Unless the ID can be considered semi-private?

```suggestion
                raise Exception(f"tree id not in store: {tree_id.hex()}")
```